SU-HSDO/suhumsci
# SHS-60: Align view “more link” buttons to the right

# READY FOR REVIEW
https://fourkitchens.atlassian.net/browse/SHS-60 

## Summary
Aligns all "more links" added by the ViewUI to the right-hand side of the container

## Need Review By (Date)
unknown

## Urgency
low

## Steps to Test
1. Pull code, start your local lando/gitpod instance. 
2. Go to https://hs-colorful.suhumsci.loc/ (homepage of hs-colorful sight) and verify the "more link" for the `Spotlight` news section is aligned to the right of its container.
3. Go to https://hs-colorful.suhumsci.loc/default-views/news (default views page for hs-colorful) and verify the "more link" for the `Spotlight` news section is aligned to the right of its container.
4. Review the code changes to `humsci_basic` and verify they are global enough to affect the other sites.
